web/static/custom/bountyhub.js

```
@@ -453,26 +454,30 @@ function populateBadges(attributes) {
     const accordion = document.getElementById('assetAccordion');
+    const hackerone_handle = data.attributes.handle;
+    const program_name = data.attributes.name;
     const assetTypes = {
         WILDCARD: [], DOMAIN: [], IP_ADDRESS: [], CIDR: [], URL: []
     };
 
     data.relationships.structured_scopes.data.forEach(scope => {
         const type = scope.attributes.asset_type;
-        if (assetTypes.hasOwnProperty(type)) {
+        const eligible_for_submission = scope.attributes.eligible_for_submission;
+        if (assetTypes.hasOwnProperty(type) && eligible_for_submission) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)